inst/templates/circleci-config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: {{{ image }}}
+    steps:
+      - restore_cache:
+          keys:
+            - cache
```

Comment:
From reading https://circleci.com/docs/2.0/caching/#using-keys-and-templates I think we would want something like

```
 - r-pkg-cache-{{ arch }}-{{ .Branch }}
 - r-pkg-cache-{{ arch }}-
```

This would give you specific caches for the architecture and branch, but would fall back to caches if they are available not available.

Reply:
OK. I can update the caching logic.